Ktisis/Overlay/GizmoState.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace Ktisis.Overlay
+{
+	/* Guizmo State machine:
+							  ^
+							  |
+							  |event: OnGuizmoChange(GuizmoState)
+							  |
+							  |
+							  |
+				IsUsing()     |
+			 +----------------+---------+
```

Comment:
This graph appears to have gotten a bit out of-sync with itself after the indentation changes...